file.js

```diff
@@ -39,14 +80,15 @@ module.exports = function (file, block_size, flags) {
   return {
     get: function (i, cb) {
       offset.once(function (_offset) {
-        function onReady () {
-          var max = ~~(_offset / block_size)
-          if(i > max)
-            return cb(new Error('aligned-block-file/file.get: requested block index was greater than max, got:'+i+', expected less than or equal to:'+max))
+        var max = ~~(_offset / block_size)
+        if(i > max)
+          return cb(new Error('aligned-block-file/file.get: requested block index was greater than max, got:'+i+', expected less than or equal to:'+max))
```

Comment:
Move into `onReady()` because we don't guarantee that calling out-of-order should throw an error.